content/installation/java/package/maven-cargo.md

```diff
@@ -0,0 +1,193 @@
+
+* Create the file and open it in your preferred text editor. 
+
+* Paste in the following contents:
+
+```yaml
+api:
+  url: <contrast_url>
+  api_key: <your_api_key>
+  user_name: <agent_user_name>
+  service_key: <agent_user_service_key>
+```
+
+* Replace `<contrast_url>`, `<your_api_key>`, `<agent_user_name>` and `<agent_user_service_key>` with the values you
+obtained from the Contrast UI in the previous step, and save the file.
+
+<details><summary><b>Configuration note</b></summary>
+<p>You can <a href="installation-javaconfig.html">configure the Contrast Java agent</a> using a file, Java system properties or environment variables. For developers, Contrast recommends putting only the variables that are shared across multiple applications (e.g., credentials and Contrast UI connection details) in this
+file. That way you can manage them in one place. <b>Step 9</b> shows you how to set application-specific configurations.</p></details>
+
+## Step 3: Clone the examples repository
+
+* Open a command prompt, and run the following command to clone the examples repository:
+
+```console
+$ git clone https://github.com/Contrast-Security-OSS/contrast-java-examples.git
+Cloning into 'contrast-java-examples'...
+remote: Enumerating objects: 12, done.
+remote: Counting objects: 100% (12/12), done.
+remote: Compressing objects: 100% (9/9), done.
+remote: Total 12 (delta 1), reused 9 (delta 1), pack-reused 0
+Unpacking objects: 100% (12/12), done.
+```
+
+* Then make your working directory the root of the `maven-cargo` example:
+
+```console
+cd contrast-java-examples/maven-cargo
+```
+
+The rest of these instructions are written in the context that you've kept your command prompt open and the working directory is the root of the `maven-cargo` example.
+
+## Step 4: Test and build the project
+
+* Check to make sure everything starts off in a working state by running the integration test. 
+
+The `maven-cargo-plugin` starts a Jetty server on port 8080, and the `EchoServletIT` integration test verifies the behavior of the `EchoServlet` by sending an HTTP request to the Jetty server. Finally, the `maven-cargo-plugin` shuts down the Jetty server.
+
+```console
+$ ./mvnw clean verify
+[INFO] Scanning for projects...
+[INFO]
+[INFO] ---------< com.contrastsecurity.examples:contrast-maven-cargo >---------
+[INFO] Building Contrast Maven Cargo Example 1.0
+[INFO] --------------------------------[ war ]---------------------------------
+[INFO]
+    ... omitting some output ...
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  3.467 s
+[INFO] Finished at: 2019-03-21T22:57:22-04:00
+[INFO] ------------------------------------------------------------------------
+```
+
+> **Note:** On Windows, run `mvnw.cmd clean verify` instead.
```

Comment:
It will be obvious. I expect Windows developers will come to this conclusion by themselves; however, I think we should keep this here just to let them know we're thinking of them 😄